bibcat/utils/run_bibcat_serial.sh

```diff
@@ -11,22 +11,23 @@ END_INDEX=43
 
 
 # Directory for logs
-LOG_DIR="/home/jyoon/bibcat/logs"
+LOG_DIR="/your/log/dir/path"
```

Comment:
As much as possible, I think we want to have code/scripts executable as is out-of-the-box, and minimize the level of manual instruction.   

If you want to retain the user flexibility here, I would make the logs directory and the batch file directory path command line arguments using `argparse`, and move all this code into a function that's called in the `__main__` namespace.

Reply:
This is a good idea. Let me think about this.

Reply:
I made some updates on the Bash script to allow setting batch directory as command line argument. I want to keep it as it for now since you are implementing BatchAPI. I may revisit the argparse option in the future depending on needs. I want to focus on run evaluation of the historic run results for now.